src/__tests__/downshift.get-item-props.js

```diff
@@ -161,32 +167,74 @@ test(`getItemProps doesn't include event handlers when disabled is passed (for I
   }
 })
 
+test(`disabled item can't be selected by pressing enter`, () => {
+  const items = [
+    {item: 'Chess', disabled: true},
+    {item: 'Dominion', disabled: true},
+    {item: 'Checkers', disabled: true},
+  ]
+  const utils = renderDownshift({items})
+  const {input, arrowDownInput, enterOnInput, changeInputValue} = utils
+
+  const firstItem = utils.queryByTestId('item-0')
+  expect(firstItem.hasAttribute('disabled')).toBe(true)
+
+  // input.simulate('keydown')
+  changeInputValue('c')
+  // ↓
+  arrowDownInput()
+  // ENTER to select the first one
+  enterOnInput()
+  // item was not selected -> input value should still be 'c'
+  expect(input.value).toBe('c')
+})
+
 function renderDownshift({
-  items = ['Chess', 'Dominion', 'Checkers'],
+  items = [{item: 'Chess'}, {item: 'Dominion'}, {item: 'Checkers'}],
   props,
 } = {}) {
-  const renderSpy = jest.fn(({getItemProps}) => (
+  const renderSpy = jest.fn(({getItemProps, getInputProps}) => (
     <div>
+      <input {...getInputProps({'data-testid': 'input'})} />
```

Comment:
What's the purpose of this `data-testid`?

Reply:
Later on in this function I query for the input element:
‘const input = utils.queryByTestId('input')‘

Reply:
Ah, I missed that. Thanks!